### PR TITLE
Bug fixing

### PR DESCRIPTION
**Pull request by kiraigg**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-USER_PASS = sportnewsREACT
-SECRET_KEY = SECRET
-RESET_PASSWORD_URL = https://secret-retreat-50026.herokuapp.com/resetpassword
-CLOUDINARY_URL=cloudinary://474762673622788:qG-1vpuxxusl_yvSIn4do1TkMqo@diirtdais
+DATABASE_URL=
+USER_EMAIL =
+USER_PASS =
+SECRET_KEY =
+RESET_PASSWORD_URL =
+CLOUDINARY_URL=
```
Comment (forenZZy) [inline .env]: Не чіпай

Comment (anna-kuzii) [inline .env]: pls leave it, as heroku will be broken

File: client/src/components/Menu/DesktopMenu/MenuItem/index.jsx [hunk 2]
```diff
@@ -13,20 +14,19 @@ export class MenuItem extends Component {
 
   render() {
     const { index, activeItem } = this.props,
-      { item: { title, menu } } = this.props,
+      { item: { title, menu, path } } = this.props,
```
Comment (anna-kuzii) [inline client/src/components/Menu/DesktopMenu/MenuItem/index.jsx]: where do u take it? can't find

File: client/src/components/Menu/DesktopMenu/style.scss [hunk 3]
```diff
@@ -109,17 +118,25 @@
       left: 100%;
       margin-top: 0;
 
+      .home-link {
+        display: none;
+      }
+
       .active-menu {
         display: none;
```
Comment (anna-kuzii) [inline client/src/components/Menu/DesktopMenu/style.scss]: combine two classes

File: client/src/components/Menu/MobileMenu/index.jsx [hunk 4]
```diff
@@ -63,9 +63,11 @@ export class Menu extends Component {
...
       toggleIcon = activeMenu ? burgerIcon : closeMenu;
 
-    (activeMenu)
-      ? document.body.style.overflow = 'visible'
-      : document.body.style.overflow = 'hidden';
+    if (activeMenu) {
+      document.body.style.overflow = 'visible';
+    } else {
+      document.body.style.overflow = 'hidden';
+    }
```
Comment (anna-kuzii) [inline client/src/components/Menu/MobileMenu/index.jsx]: why do u change it?

File: client/src/containers/UpdateProfile/component/index.jsx [hunk 5]
```diff
@@ -11,12 +11,12 @@ class UpdateProfile extends Component {
...
-    const users = this.props.user;
+    const { user } = this.props;
 
     this.state = {
-      email: users.email,
-      newEmail: users.email,
-      newName: `${users.firstName} ${users.lastName}`,
+      email: user.email,
+      newEmail: user.email,
+      newName: `${user.firstName}  ${user.lastName}`,
```
Comment (anna-kuzii) [inline client/src/containers/UpdateProfile/component/index.jsx]: use destr

File: client/src/components/Menu/DesktopMenu/MenuItem/index.jsx (hunk 2 above)
Comment (kiraigg) [reply client/src/components/Menu/DesktopMenu/MenuItem/index.jsx]: menuItems, json

**Pull request merged**